Content.Shared/Psionics/PsionicComponent.cs

```diff
@@ -1,20 +1,137 @@
-using Content.Shared.Actions;
+using Content.Shared.Psionics;
 using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
 
 namespace Content.Shared.Abilities.Psionics
 {
     [RegisterComponent, NetworkedComponent]
     public sealed partial class PsionicComponent : Component
     {
-        public EntityUid? PsionicAbility = null;
-
         /// <summary>
         ///     Ifrits, revenants, etc are explicitly magical beings that shouldn't get mindbreakered.
```

Comment:
```suggestion
        ///     Ifrits, revenants, etc. are explicitly magical beings that shouldn't get mindbreakered.
```
I didn't change mindbreakered cause I thought it was funny.